vhdbuilder/packer/init-variables.sh

```diff
@@ -36,35 +43,6 @@ if [ -z "$rg_id" ]; then
 	az group create --name $AZURE_RESOURCE_GROUP_NAME --location ${AZURE_LOCATION}
 fi
 
-if [ -n "${VNET_RESOURCE_GROUP_NAME}" ]; then
-	VIRTUAL_NETWORK_NAME="vnet"
-	VIRTUAL_NETWORK_SUBNET_NAME="subnet"
-	NETWORK_SECURITY_GROUP_NAME="nsg"
-
-	echo "creating resource group ${VNET_RESOURCE_GROUP_NAME}, location ${AZURE_LOCATION} for VNET"
-	az group create --name ${VNET_RESOURCE_GROUP_NAME} --location ${AZURE_LOCATION} \
-		--tags 'os=Windows' 'createdBy=aks-vhd-pipeline' 'SkipASMAzSecPack=True'
```

Comment:
Do we need these tags on every resource in the network stack? Packer is already configured to tag the build VM with these tags as is.